src/taco-cli/cli/utils/cordovaWrapper.ts

```diff
@@ -188,6 +190,24 @@ class CordovaWrapper {
         options: { logLevel?: tacoUtility.InstallLogLevel, isSilent?: boolean, captureOutput?: boolean } = {}): Q.Promise<T | string> {
         return CordovaHelper.tryInvokeCordova<T | string>(apiFunction, () => CordovaWrapper.cli(processArgs(), options.captureOutput), options);
     }
+
+    /**
+     * Returns a path to the local Cordova command if ran from a taco project, otherwise it will return the
+     * global Cordova command.
+     */
+    private static getCordovaCommand(): Q.Promise < string > {
+        return CordovaHelper.tryInvokeCordova<string>((cordova: typeof Cordova): string => {
+        return (<{ packageTargetPath?: string }> cordova).packageTargetPath;
```

Comment:
no I am referring to Path.join here.. the logic to get to the executable should be part of TacoPackageLoader.

instead of exposing T.packageTargetPath, we should expose T.binPath or T.execPath.

Makes sense?